sympy/printing/fcode.py

```diff
@@ -546,6 +574,11 @@ def fcode(expr, assign_to=None, **settings):
         Note that currently the only distinction internally is between
         standards before 95, and those 95 and after. This may change later as
         more features are added.
+    name_mangling : bool, optional
+        If True, then variables (which would have been identical in
+        case-insensitive Fortan) are mangled by appending different number
```

Comment:
It's Fortran, not Fortan. (I corrected that elsewhere). Parentheses don't quite work here. I suggest "If True, then the variables that would become identical in case-insensitive Fortran are..."